source/overview/security.rst

```diff
@@ -64,13 +64,13 @@ Authentication Safeguards
 Access Control Policy
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If a user logins in and enters the correct email address but wrong password a "your password is incorrect" response is presented. This can allow email/password enumeration. This is a deliberate design decision we've made for the product in the context of overall system security. 
+If a user logins in and enters the correct email address but wrong password a "your password is incorrect" response is presented. This can allow account enumeration. This is a deliberate design decision we've made for the product in the context of overall system security. 
```

Comment:
nit: "account enumeration" here but "enumerating user accounts" in the next paragraph. This might be by design, given the context of `user` in the previous sentence, but just surfacing.

Reply:
Good catch, thank you!